packages/protocol/lib/compatibility/version.ts

```diff
@@ -26,7 +34,18 @@ export class ContractVersion {
       Number(v[3])
     )
   }
-  
+
+  static fromBuffer = (version: Buffer): ContractVersion => {
```

Comment:
Maybe add some docs explaining this. Without context I'm not sure what kind of buffer is this about.